clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -158,6 +166,15 @@ record_filter_t::record_filter_t(
     , stop_timestamp_(stop_timestamp)
     , verbosity_(verbose)
 {
+    /* Check if encodings2regdeps filter is present.
```

Comment:
See below: I think this should be removed in favor of a general filetype interface.

Reply:
Done.